.travis.yml

```diff
@@ -0,0 +1,55 @@
+language: go
+go:
+  - 1.6
+env:
+  - "MY_GOOS=linux MY_GOARCH=amd64"
+  - "MY_GOOS=darwin MY_GOARCH=amd64"
+  - "MY_GOOS=windows MY_GOARCH=amd64"
+  - "MY_GOOS=windows MY_GOARCH=386"
+
+before_install:
+  #we actually use 1.7 which isn't supported yet
+  #by downloading it in this section
+  
+  - curl -o go.tar.gz -sL https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz
+  - tar -C $HOME -xf go.tar.gz
+  - rm go.tar.gz
+  - export GOROOT="${HOME}/go"
+  - export PATH="${GOROOT}/bin:${PATH}"
+  - export GOOS="${MY_GOOS}"
+  - export GOARCH="${MY_GOARCH}"
+  - go version
+  - go env
+  #end of download
+
+  - go get -u -v github.com/alecthomas/gometalinter
+  - go get code.google.com/p/go.tools/cmd/cover
```

Comment:
It feels like this is failing because the correct URL is [golang.org/x/tools/cmd/cover](golang.org/x/tools/cmd/cover)
You might want to give a look at the [updated `.travis.yml` file](https://github.com/taskcluster/taskcluster-worker/blob/master/.travis.yml) in the [taskcluster-worker repo](https://github.com/taskcluster/taskcluster-worker)